backend/Dockerfile.cloud

```diff
@@ -0,0 +1,109 @@
+FROM python:3.11.7-slim-bookworm
```

Comment:
A lot of the comments and things are not correct. I would say this just should not be used by the community.